src/main/java/org/fogbowcloud/saps/engine/core/dispatcher/SubmissionManagerImpl.java

```diff
@@ -19,50 +11,79 @@
 import org.restlet.representation.Representation;
 import org.restlet.resource.ClientResource;
 
+import java.io.IOException;
+import java.util.*;
+import java.util.stream.Collectors;
+
 /**
  * Concret implementation of {@link SubmissionManager}.
  */
 public class SubmissionManagerImpl implements SubmissionManager {
 
     private static final Logger LOGGER = Logger.getLogger(SubmissionManagerImpl.class);
 
-    // TODO: Replace static string by config txt file
-    private static final String REMOTE_INSTANCE_URL = "";
+    private static final String SAPS_NEIGHBORS_URLS = "saps_neighbors_urls";
 
+    private Properties properties;
     private SubmissionDispatcher submissionDispatcher;
 
-    public SubmissionManagerImpl(SubmissionDispatcher submissionDispatcher) {
+    public SubmissionManagerImpl(Properties properties, SubmissionDispatcher submissionDispatcher) {
+        this.properties = properties;
         this.submissionDispatcher = submissionDispatcher;
     }
 
     @Override
-    public List<Task> addTasks(SubmissionParameters submissionParameters) throws IOException, ParseException, SQLException {
-        List<ImageTask> processedTasks = getRemotelyProcessedTasks(submissionParameters);
-        for (ImageTask processedTask: processedTasks) {
-            processedTask.setState(ImageTaskState.REMOTELY_ARCHIVED);
+    public List<Task> addTasks(SubmissionParameters submissionParameters) {
```

Comment:
is there any place (before or after this call) where we log the submissionParameters?

Reply:
Yes, they're logged before at `ImageResource#insertTasks`